frontend/package.json

```diff
@@ -57,10 +57,11 @@
   "scripts": {
     "start": "webpack serve --mode development --host 0.0.0.0",
     "build": "webpack --mode production ./src/main/react",
-    "lint": "prettier --check './src/main/react/**/*.js'",
-    "lint:fix": "prettier --write './src/main/react/**/*.js' --fix",
+    "lint": "prettier --check './src/main/react/**/*.tsx'",
+    "lint:fix": "prettier --write './src/main/react/**/*.tsx' --fix",
+    "lint-staged": "lint-staged",
```

Comment:
'fixed' prettier config, as it was still linting js files, rather than `ts(x)` files.  leftover from https://github.com/ZzAve/teambalance/pull/175)